app/parse.py

```diff
@@ -8,5 +13,47 @@ class Course:
     duration: str
 
 
+def parse_single_course(course: BeautifulSoup) -> Course:
+    return Course(
+        name=course.select_one(
+            "h1[data-qa='profession-title']"
+        ).text.split(":")[0],
+        short_description=course.select_one(
+            "p.typography_landingTextMain__Rc8BD."
+            "SalarySection_aboutProfession__1VFHK"
+        ).text,
```

Comment:
Similar to the previous comment, this CSS selector is tightly coupled with the current HTML structure of the website. It might fail if the website's structure changes. Consider adding error handling for robustness.